requirements.txt

```diff
@@ -311,7 +311,7 @@ pyvips==2.2.3
     # via large-image-source-vips
 rcssmin==1.1.2
     # via django-compressor
-redis==5.2.0
+redis==5.2.1
```

Comment:
Is this an error ? Why change this requirement ? This file should not be update manually.

Reply:
Désolée, pour le coup, je ne sais pas ce qu'il s'est passé. J'ai remis le fichier requirements.txt comme il est sur la branche master.